zzzz_modular_occulus/code/game/machinery/bsd.dm

```diff
@@ -2,25 +2,61 @@
 //This thing is spoiler heavy. Do not map this or use it WITHOUT MY PERMISSION -Ikky
 //Unless you're downstream then I can't stop you.
 
-/obj/structure/bsd
+/obj/machinery/bsd
 	name = "A Runtime Error"
 	desc = "One day you will have to answer for your actions, and God may not be so... forgiving."
 	icon = 'zzzz_modular_occulus/icons/obj/bsd.dmi'
 	icon_state = "faceofgod"
 	density = TRUE
 	anchored = TRUE
+	use_power = NO_POWER_USE
+	unacidable = 1
 
-/obj/structure/bsd/New() //Don't look at it
-	START_PROCESSING(SSobj, src)
+/obj/machinery/bsd/ex_act(severity, target) //immune to explosions. Make this do very bad things later.
+	return
+
+/obj/machinery/bsd/New() //Don't look at it
+	//START_PROCESSING(SSobj, src)
+	bound_width = world.icon_size*2
+	bound_height = world.icon_size*2
 	..()
 
-/obj/structure/bsd/Process()
+/obj/machinery/bsd/Process()
 	for(var/mob/living/carbon/human/H in oviewers(5, src))
-		H.sanity.insight += 0.05
-		H.sanity.level -= 30
-		H.sanity.environment_cap_coeff = 0
-		H.apply_effect(100, IRRADIATE)
-		H.adjust_hallucination(120, 30)
-
-/obj/structure/bsd/Destroy()
-	STOP_PROCESSING(SSobj, src)
+		H.apply_effect(30, IRRADIATE) //Radiation suits should protect from this.
+		if(H.glasses && istype(H.glasses, /obj/item/clothing/glasses/powered))
+			if(prob(1))
+				if(prob(1))
+					to_chat(H, "You feel uneasy.")
```

Comment:
Consider adding a `SPAN` to this. SPAN_WARNING or SPAN_DANGER, perhaps

Reply:
Can be added later. This isn't the first pass I'm doing on this.